app/app.go

```diff
@@ -198,6 +201,7 @@ func NewXrnApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bo
 	app.crisisKeeper = crisis.NewKeeper(crisisSubspace, invCheckPeriod, app.distrKeeper,
 		app.bankKeeper, app.feeCollectionKeeper)
 	app.upgradeKeeper = upgrade.NewKeeper(app.upgradeStoreKey, app.cdc)
+	app.upgradeKeeper.SetUpgradeHandler("regen-test-1000-upgrade-1", func(ctx sdk.Context, plan upgrade.Plan) { })
```

Comment:
I would add a comment here, saying this is a checkpoint for code upgrade - in particular PR #66 (powerreduction change)